plan/planbuilder.go

```diff
@@ -878,8 +890,8 @@ func (b *planBuilder) buildInsert(insert *ast.InsertStmt) Plan {
 			return nil
 		}
 		// Check set list contains generated column or not.
-		if columnByName[assign.Column.Name.L].IsGenerated() {
```

Comment:
Does it panic here?

Reply:
If it does panic here, because the assign.Column.Name.L is not in `columnByName`, I think, we can just skip it here( check whether the value is null is ok). No need to add checkGeneratedColumn function to iterator the columns array every time.

Reply:
I don't think it is a good solution to skip it directly. We should not use `columnByName` to avoid potential bugs.

Reply:
This is not potential bugs, actually it is expected that there is some inserting column not in columnByName. We should not reduce the performance here. @jackysp

Reply:
line 796 check the situation, but here not, so it panic.